elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "PaackEng/paack-ui",
     "summary": "Paack's Design System applied over Elm UI",
     "license": "BSD-3-Clause",
-    "version": "4.0.0",
+    "version": "5.0.0",
```

Comment:
![Oshino Meme](https://i.imgflip.com/5dhcy2.jpg)

Reply:
![image](https://user-images.githubusercontent.com/14297772/122244187-0db2b380-ce9b-11eb-846f-47895bd0179b.png)

Reply:
Hey @renanpvaz , now that we're on this subject, `RadioSize` should looked like:
- https://github.com/PaackEng/elm-google-maps/blob/ed273d827f549459297a497b0e41f512ef78b529/src/GoogleMaps/Map.elm#L91
- https://github.com/PaackEng/elm-google-maps/blob/ed273d827f549459297a497b0e41f512ef78b529/src/GoogleMaps/Map.elm#L172
- https://github.com/PaackEng/elm-google-maps/blob/ed273d827f549459297a497b0e41f512ef78b529/src/GoogleMaps/Map.elm#L188